src/main/java/seedu/address/logic/parser/TimeParser.java

```diff
@@ -50,7 +50,7 @@ public static AppointmentTime parse(String args) throws ParseException {
         if (matchToday.matches()) {
             LocalDate today = LocalDate.now();
             String todayString = today.format(DATE_FORMAT);
-            return new AppointmentTime(todayString + args.substring(5));
+            return new AppointmentTime(todayString + args.split(" ")[1]);
```

Comment:
split the args to extract the hours, because length of today can vary

Reply:
`args.split(" ")[1]` be trimmed as well.

Reply:
add a space after `todayString`